client/src/views/CellView.tsx

```diff
@@ -111,141 +121,148 @@ const CellView = () => {
         confirmReport={confirmReport}
         confirmSacrifice={confirmSacrifice}
         confirmReset={confirmReset}
+        bruteList={bruteList}
+
       />
     )
     : (
-      <Page title={`${brute.name} ${t('MyBrute')}`} headerUrl={`/${brute.name}/cell`}>
```

Comment:
You should revert this component structure change and include the icons inside the Page component, as to not add unnecessary react fragments.

Reply:
But if I put in this component it will be visible for all pages and we want it on the cell view only no ?

Reply:
I meant to leave it inside the CellView, but not outside the <Page>, inside